projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/order-amend.service.ts

```diff
@@ -0,0 +1,128 @@
+import { formatCurrency, getCurrencySymbol } from '@angular/common';
+import { Injectable } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Order, OrderEntry, Price } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { map, switchMap, tap } from 'rxjs/operators';
+import { OrderDetailsService } from '../order-details/order-details.service';
+import { OrderAmendType } from './order-amend.model';
+
+function ValidateQuantity(control: FormControl) {
```

Comment:
If this can be re-used we have a `custom-form-validators.ts` where this could go

Reply:
I believe it's very specific, i'll keep it at the current location. Ideally we would introduce customisations on the form validations in the future.